bliss/simulator/prior.py

```diff
@@ -228,12 +249,14 @@ def _load_color_distribution(self, sdss_fields):
 
```

Comment:
I didn't notice this before, but are we just using the "psfflux" column to sample the ratios? IIRC "psfflux" is the modeled star fluxes - we need to look at "cmodelflux" for galaxies, and then pick depending on the object type.

Reply:
Yeah "psfflux" is used for the nanomaggy count for stars and galaxies. If that's not actually the case for galaxies then that should definitely be changed.

Reply:
Yeah, that's definitely a problem then. See how it's done in PhotoFullCatalog - we should do the same thing here.
https://github.com/prob-ml/bliss/blob/f1b7fcf23873c0e9985453b939dcd05a10b96814/bliss/surveys/sdss.py#L276-L282

Reply:
Added!